Rakefile

```diff
@@ -14,11 +14,32 @@ Rake::TestTask.new do |t|
   t.pattern = "spec/*_spec.rb"
 end
 
+DATA_DEST_PATH   = File.expand_path('../source/data/awards.json', __FILE__)
+DATA_SRC_PATH    = File.expand_path('../data/awards.json', __FILE__)
+INJECT_DEST_PATH = File.expand_path('../data/injectable.yml', __FILE__)
```

Comment:
If I see this file in the data directory, I'd have no idea what it is. How about naming it `preloaded_awards.yml` or something similar?